src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileWriterImpl implements FileWriter {
```

Comment:
The FileWriterImpl class should not begin with an empty line. Please remove the redundant empty line at the start of the class definition.